templates/accel-ppp.j2

```diff
@@ -299,8 +299,8 @@ pubchan=accel-ppp
 # select the event types to emit a message via redis
 #
 ev_ses_starting=yes
-#ev_ses_finishing=yes
```

Comment:
this should be mentioned and explained in the commit message

Reply:
Ok.